src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Release.nuspec

```diff
@@ -12,8 +12,8 @@
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4. </summary>
     <description>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4. Targets .NET 4.5 and .NET Platform Standard 1.1.
-OData .NET library is open source at http://github.com/OData/odata.net. Documentation for the library can be found at https://odata.github.io/odata.net.</description>
-    <releaseNotes>http://odata.github.io/odata.net/v7/#ODL-$VersionFullSemantic$</releaseNotes>
+OData .NET library is open source at http://github.com/OData/odata.net. Documentation for the library can be found at https://docs.microsoft.com/en-us/odata/.</description>
+    <releaseNotes>https://docs.microsoft.com/en-us/odata/changelog/$VersionFullNumberRelease$</releaseNotes>

```

Comment:
>https://docs.microsoft.com/en-us/odata/changelog/$VersionFullNumberRelease$ [](start = 18, length = 75)

URL should be of the form: [https://docs.microsoft.com/en-us/odata/changelog/odatalib-7x#odatalib-781-release](https://docs.microsoft.com/en-us/odata/changelog/odatalib-7x#odatalib-781-release). Does $VersionFullNumberRelease$ resolve to "/odatalib-7x#odatalib-781release"?